.circleci/config.yml

```diff
@@ -56,64 +53,72 @@ workflows:
           name: test-linux-arm64
           executor: linux-arm64
           requires:
-            - test-build-arm64-linux
+            - test-build-linux-arm64
       - test-prebuilt:
           name: test-darwin
           executor: darwin
           requires:
-            - test-build-amd64-darwin
+            - test-build-darwin-amd64
       - test-prebuilt:
           name: test-windows
           executor: windows
           requires:
-            - test-build-amd64-windows
-      - fluxtest:
-          requires:
-            - godeps
-      - tlstest:
-          requires:
-            - godeps
-      - lint:
-          # TODO: Remove this alias as part of https://github.com/influxdata/influxdb/issues/22623
-          name: golint
+            - test-build-windows-amd64
+      - fluxtest
+      - tlstest
+      - lint
       - build:
+          name: build-snapshot-<< matrix.os >>-<< matrix.arch >>
+          build-type: snapshot
+          matrix:
+            parameters:
+              os: [ linux, darwin, windows ]
+              arch: [ amd64, arm64 ]
+            exclude:
+              - os: darwin
+                arch: arm64
+              - os: windows
+                arch: arm64
+      - package-and-release:
+          name: package-snapshot
+          build-type: snapshot
+          filters:
+            branches:
+              ignore: /pull\/[0-9]+/
           requires:
-            - godeps
-      - cross_build:
-          requires:
-            - godeps
+            - build
       - test-downgrade:
           requires:
-            - build
+            - build-snapshot-linux-amd64
       - e2e-monitor-ci:
           filters:
             branches:
               ignore: /pull\/[0-9]+/
           requires:
-            - build
+            - build-snapshot-linux-amd64
       - pkg_run_test:
```

Comment:
The mix of `hyphen-case` and `snake_case` conventions in the job names here has annoyed me for awhile - since we're going to request changing the list of required jobs anyways, I could also normalize the convention now. My preference would be to go with `hyphen-case` for everything

Reply:
+100

Reply:
Done